DataHub/Operation/03_ManageHTMLTemplates.md

```diff
@@ -0,0 +1,296 @@
+[!!User interface HTML templates](../UserInterface/02j_HTMLTemplates.md)
+
+# Manage the HTML templates
+
+Define HTML templates that are used for the ETL mapping, especially for the receipt export.
+HTML templates allow you to include HTML text or Smarty templates and thus create complex queries with many logical inputs.
+Further, you are able to include certain data from all available entities using the HTML templates.
```

Comment:
daten kann man verwenden von der source-entity im html sowie alle sub-entitäten sofern sie in der json-darstellung der quell-entity vorhanden sind